runtime/pkg/email/templates/scheduled_report.mjml

```diff
@@ -40,7 +46,7 @@
 
         <mj-spacer height="20px" />
         
-        <mj-text align="center" font-size="12px">
+        <mj-text css-class="{{ if not .EditLink }}hidden{{ end }}" align="center" font-size="12px">
           To edit or unsubscribe from this report, <a href="{{ .EditLink }}">click here</a>.
         </mj-text>
```

Comment:
This is annoying, but we probably still need to find a way to let anonymous recipients unsubscribe from the report.

I guess this could work by setting up an unsubscribe page that will call the `UnsubscribeReport` RPC with the report token (and making the `UnsubscribeReport` RPC support calls with the report token)?

Reply:
Modified `UnsubscribeReport` to accept magic token. Now a new page need to be added at `[organization]/[project]/-/reports/[report]/unsubscribe` which will call this rpc with magic token present as query param, the rpc also expects the email of the recipient which will also be present as query param. cc @AdityaHegde

Reply:
Found one issue, using magic token one can remove any other email from the list. Need to store info about the magic token recipient or encode email.

Reply:
Fixed - matching the magic token id for report token issued for this email